functions/Get-DbaDatabase.ps1

```diff
@@ -1,12 +1,15 @@
 ﻿function Get-DbaDatabase {
-	<#
+    <#
 		.SYNOPSIS
 			Gets SQL Database information for each database that is present in the target instance(s) of SQL Server.
 
 		.DESCRIPTION
 			The Get-DbaDatabase command gets SQL database information for each database that is present in the target instance(s) of
 			SQL Server. If the name of the database is provided, the command will return only the specific database information.
 
+			We also added a number of useful columns by default such as the Last used read and write Times for the Database using 
```

Comment:
While it definitely is a useful addition to have extra properties and filtering options, this should not be a default behaviour.
Like a lot of functions, we get the SMO objects, fast and lean. Only when the user asks for more, we add functionality (and cost and time and risk).
There should be a switch to ask for this info, and without it, the query should not be executed.
@SQLDBAWithABeard can you make that like @niphlod did for the `-NoBackup` ?